### PR TITLE
Shop 29 section banner

### PR DESCRIPTION
**Pull request by sprajams**
- create section banner component
- optional subtitle and action (currently a placeholder) fields

### CODE REVIEW

File: src/components/SectionBanner/SectionBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  heading,
+  subtitle,
+  action,
+}: SectionBannerProps) => {
+  const spaceBetween = !!subtitle && !!action;
+  const noOptions = !subtitle && !action;
+
+  return (
+    <div className={styles.container}>
+      <h2 className={clsx(styles.main, styles.wordBreak)}>{heading}</h2>
```
Comment (scottqmn) [inline src/components/SectionBanner/SectionBanner.tsx]: Remove h2 and h4 headings

**Review by scottqmn — CHANGES_REQUESTED** [review]
Looks good! just one comment

File: src/components/SectionBanner/SectionBanner.tsx (hunk 1 above)
Comment (sprajams) [reply src/components/SectionBanner/SectionBanner.tsx]: Add dynamic tags in place of removing heading tags completely

**Review by scottqmn — CHANGES_REQUESTED** [review]
Same comments as #28 regarding dynamic tags

**Review by scottqmn — APPROVED** [review]

**Pull request merged**